bundles/org.openhab.binding.paradoxalarm/src/main/java/org/openhab/binding/paradoxalarm/internal/discovery/ParadoxDiscoveryService.java

```diff
@@ -0,0 +1,117 @@
+/**
+ * Copyright (c) 2010-2019 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.binding.paradoxalarm.internal.discovery;
+
+import static org.openhab.binding.paradoxalarm.internal.handlers.ParadoxAlarmBindingConstants.*;
+
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+import org.eclipse.smarthome.config.discovery.AbstractDiscoveryService;
+import org.eclipse.smarthome.config.discovery.DiscoveryResult;
+import org.eclipse.smarthome.config.discovery.DiscoveryResultBuilder;
+import org.eclipse.smarthome.core.thing.ThingUID;
+import org.openhab.binding.paradoxalarm.internal.communication.IParadoxCommunicator;
+import org.openhab.binding.paradoxalarm.internal.exceptions.ParadoxRuntimeException;
+import org.openhab.binding.paradoxalarm.internal.handlers.ParadoxAlarmBindingConstants;
+import org.openhab.binding.paradoxalarm.internal.handlers.ParadoxIP150BridgeHandler;
+import org.openhab.binding.paradoxalarm.internal.model.ParadoxInformation;
+import org.openhab.binding.paradoxalarm.internal.model.ParadoxPanel;
+import org.openhab.binding.paradoxalarm.internal.model.Partition;
+import org.openhab.binding.paradoxalarm.internal.model.Zone;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * The {@link ParadoxDiscoveryService} is responsible for discovery of partitions, zones and the panel once bridge is
+ * created.
+ *
+ * @author Konstnatin Polihronov - Initial Contribution
+ */
+public class ParadoxDiscoveryService extends AbstractDiscoveryService {
+
+    private final Logger logger = LoggerFactory.getLogger(ParadoxDiscoveryService.class);
+
+    private ParadoxIP150BridgeHandler ip150BridgeHandler;
+
+    public ParadoxDiscoveryService(ParadoxIP150BridgeHandler ip150BridgeHandler) {
+        super(ParadoxAlarmBindingConstants.SUPPORTED_THING_TYPES_UIDS, 15, false);
```

Comment:
You can omit the prefix `ParadoxAlarmBindingConstants` as you already defined it as a static import.